runauto_nag_hourly_py2.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+set -e
+
+PATH_SCRIPT="$( cd "$(dirname "$0")" ; pwd -P )"
```

Comment:
What about moving that into a runauto_nag_common.sh to avoid duplication?
to export PYTHONPATH=.
can be probably called with a source runauto_nag_common.sh